sfge/src/main/java/com/salesforce/rules/RemoveUnusedMethod.java

```diff
@@ -97,10 +98,6 @@ public List<MethodVertex> getEligibleMethods(GraphTraversalSource g) {
                                 .hasLabel(NodeType.METHOD)
                                 // The "<clinit>" method is ineligible.
                                 .has(Schema.NAME, P.neq("<clinit>"))
-                                // TODO: FOR NOW, WE'RE IGNORING CONSTRUCTORS TO CUT DOWN ON NOISE.
-                                //       IN THE FULLNESS OF TIME, AS WE FIX RELEVANT BUGS, WE'LL
-                                //       REMOVE THIS RESTRICTION.
-                                .has(Schema.NAME, P.neq("<init>"))
```

Comment:
Note that when all this line was introduced (which i'm now deleting) the ConstructorTest file should have started failing... but it didn't because the assertion inside of BaseUnusedMethodTest was not robust. So the first thing I did was fix this assertion.. .making the tests fail in ConstructorTest and then I removed this line of code to make them pass again. This allowed me to stick with my test driven development workflow (even though i didn't explicitly have to add a test).